docs/en/observability/elastic-entity-model.asciidoc

```diff
@@ -0,0 +1,67 @@
+[[elastic-entity-model]]
+= Elastic Entity Model
+
+preview::[]
+
+The Elastic Entity Model consists of:
+
+* a data model and related entity indices
+* an Entity Discovery Framework, which consists of {ref}/transforms.html[transforms] and {ref}/ingest.html[Ingest pipelines] that read from signal indices and write data to entity indices
+* a set of management APIs that empower entity-centric Elastic solution features and workflows
+
+In the context of Elastic Observability,
+an _entity_ is an object of interest that can be associated with produced telemetry and identified as unique.
+Note that this definition is intentionally closely aligned to the work of the https://github.com/open-telemetry/oteps/blob/main/text/entities/0256-entities-data-model.md#data-model[OpenTelemetry Entities SIG].
+Examples of entities include (but are not limited to) services, hosts, and containers.
+
+The concept of an entity is important as a means to unify observability signals based on the underlying entity that the signals describe.
+
+[NOTE]
+====
+* The Elastic Entity Model currently supports the <<new-experience-services,new service inventory experience>> limited to service-based entities (as identified by `service.name`) located in data identified by `logs-*` and `trace-*` index patterns
```

Comment:
data identified by `logs-*` and `filebeat*`. There's also `metrics-apm.service_transaction.1m*` but it is used for metrics aggregation, although services available in the metrics but not in logs will still be extracted (see [relevant code](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/entity_manager/server/lib/entities/built_in/services.ts#L30))